parachain/runtime/interlay/src/lib.rs

```diff
@@ -1446,7 +1445,7 @@ mod benches {
 
         // Other
         [cumulus_pallet_xcmp_queue, XcmpQueue]
-        [frame_system, frame_system_benchmarking::Pallet::<Runtime>]
+        // [frame_system, frame_system_benchmarking::Pallet::<Runtime>]
```

Comment:
Benchmarking fails for `frame_system`, need to intialize set code, something like [here](https://github.com/mangata-finance/cumulus/blob/7bcbff3605e38fb59d6882e30f7a8a10060234d0/parachain-template/runtime/src/lib.rs#L711) but it fails with 
`function or associated item not found in Pallet<Runtime>` even after feature gating through `runtime-benchmarks`

Reply:
Have you propagated the feature through `parachain-system`? https://github.com/paritytech/cumulus/blob/d4bb2215bb28ee05159c4c7df1b3435177b5bf4e/pallets/parachain-system/src/lib.rs#L1428-L1429